Gemfile

```diff
@@ -60,6 +60,7 @@ gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sshkey",                         "~>1.8.0",       :require => false
+gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
As someone who's opinion can be ignored if you so choose, I am wondering if this is something we want to include in the Gemfile by default, and maybe it is something that folks should instead add to the Gemfile.local?

The reason I say this is we now just added two dependencies we did not need before: `daemons` and `eventmachine` (and I am unsure if they add there own as well).  Seeing as `puma` has zero extra dependencies, should we just include that, and allow other webservers to be included if the dev/user so chooses?

Related question:  if we were to want to try out `unicorn` or `passenger` (for example), would we be adding that to the `Gemfile` as well?  If we make the standard to use the Gemfile.local for this, then there would be nothing else required to support the other two in the codebase.